Scripts/Utilities/TransformCopy.cs

```diff
@@ -0,0 +1,62 @@
+﻿#if UNITY_EDITOR
+using UnityEngine;
+
+namespace UnityEditor.Experimental.EditorVR.Helpers
+{
+    sealed class TransformCopy : MonoBehaviour
+    {
+        enum Space
+        {
+            World,
+            Local
+        }
+
+        [SerializeField]
+        Transform m_SourceTransform;
+
+        [SerializeField]
+        float m_XPositionPadding = 0f;
+
+        [SerializeField]
+        float m_YPositionPadding = 0f;
+
+        [SerializeField]
+        float m_ZPositionPadding = 0f;
+
+        [SerializeField]
+        bool m_ParentUnderSource = true;
+
+        [SerializeField]
+        Space m_Space;
+
+        [SerializeField]
+        bool m_ForceAlwaysUpdate = false;
+
+        Vector3 m_Padding;
+
+        void Awake()
+        {
+            m_Padding = new Vector3(m_XPositionPadding, m_YPositionPadding, m_ZPositionPadding);
+            m_SourceTransform = m_SourceTransform ?? transform.parent;
+            if (m_ParentUnderSource)
+                transform.SetParent(m_SourceTransform, false);
+
+            DriveTransform();
+        }
+
+        void Update()
```

Comment:
I usually need LateUpdate for these kinds of scripts - where is this being used?

Reply:
This is being used on the left and right bones of the background objects

Reply:
If desired, I could add an "update method" inspector enum, allowing the user to choose a desired update stage (enum comparison in Update & LateUpdate).

Reply:
In most LazyFollow cases, I'd use LateUpdate.  Though, in this case it's a normal 1:1 timing update, ala UITransformCopy